hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -468,8 +468,11 @@ public void run() throws Exception {
         }
     }
 
-    protected void doTestGetAllReturnsFromNearCache() {
-        NearCacheConfig nearCacheConfig = createNearCacheConfig(InMemoryFormat.OBJECT);
+    protected void doTestGetAllReturnsFromNearCache(InMemoryFormat inMemoryFormat) {
```

Comment:
I parameterized this test as well, so it isn't executed twice with the same configuration (and burning CPU time).